services/prototype.go

```diff
@@ -302,7 +323,7 @@ func searchDatabase(ctx context.Context,
 	// is the database valid?
 	_, ok := config.Databases[input.Database]
 	if !ok {
-		return nil, fmt.Errorf("Database %s not found", input.Database)
+		return nil, databaseError(databases.NotFoundError{Database: input.Database})
```

Comment:
is the `_` in the line 
```
_, ok := config.Databases[input.Database]
```
a different `_` to the one in the function args?

(sorry, not enough caffeine brain)

Reply:
Same idea--an unused value.